src/abstract-view.js

```diff
@@ -345,7 +345,7 @@ Marionette.AbstractView = Backbone.View.extend({
     var prefixedEventName = eventPrefix + ':' + eventName;
     var callArgs = [this].concat(args);
 
-    Marionette._triggerMethod(layoutView, prefixedEventName, callArgs);
+    Marionette._triggerMethod(layoutView, [prefixedEventName].concat(callArgs));
```

Comment:
@megawac We added a concat back in here because Paul's win of converting `_triggerMethod()` to two arguments seemed worth it.  Plus we will be moving `_triggerEventOnParentLayout` function to `Marionette.Events` in the near future.